pypeman/commands.py

```diff
@@ -159,21 +181,35 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
     except KeyboardInterrupt:
         pass
 
-    print("End started tasks...")
-
-    # TODO: Shouldn't we also stop all endpoints?
-    # TODO: and if yes, before or after stopping the channels
-    # Intuitively, as we start endpoints after the channels we should probably stop
-    # the endpoints before te channels
+    logger.debug("loop was stopped.")
+    logger.debug("stop pypeman graph as nicely as possible")
+    # TODO: might have to:
```

Comment:
Also fill a ticket, or are you planning this work in this PR ?

Reply:
Will do.

No. I want to do this later (or as soon as it turns out to be a blocking issue, which it shouldn't), as this is just an attempt to fix something, which is already broken.

All energy is currently invested into the development of plug-in, that is already  running but still not feature complete

Reply:
ticket created and comment changed

Reply:
If interested, then PM me outside of github and I can show you the current plugin under development to give you an idea.

Reply:
Fine.